backend/services/media_generation_service.py

```
@@ -0,0 +1,421 @@
+        """Generate image using PIL (production-quality)"""
+        try:
+            # Create image
+            img = Image.new('RGB', (params['width'], params['height']), params['background_color'])
+            draw = ImageDraw.Draw(img)
+            
+            # Try to use custom font, fallback to default
+            try:
+                title_font = ImageFont.truetype("arial.ttf", 60)
+                quote_font = ImageFont.truetype("arial.ttf", 40)
+                hashtag_font = ImageFont.truetype("arial.ttf", 30)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)